docs/nima/inject/injection_intro.adoc

```diff
@@ -66,7 +76,7 @@ Once you have planned how each of your resources should support Injection, you s
 
 == API
 
-Adding Ijection behavior to your Helidon Níma application involves just a few simple steps.
+Adding Injection behavior to your Helidon Níma application involves just a few simple steps.
```

Comment:
Helidon Níma application (remove Nima ref here?)

Reply:
Fixed in the SE version. The Nima directory will be deleted so changes are made to the SE versions only.